commit 0df37528ebe495c2974e755be464722d0a8bfda1
Author: Vitaly Buka <vitalybuka@google.com>
Date:   Wed Oct 5 09:47:51 2022 -0700

    Revert "[mlir][sparse] Restore case coverage warning fix"
    
    Breaks https://lab.llvm.org/buildbot/#/builders/168/builds/9288
    
    This reverts commit 83839700c32996c58ddebc0c74e3dc4970e005bc.

diff --git a/mlir/lib/ExecutionEngine/SparseTensor/File.cpp b/mlir/lib/ExecutionEngine/SparseTensor/File.cpp
index b2632313f7d3..eac3b0278ce3 100644
--- a/mlir/lib/ExecutionEngine/SparseTensor/File.cpp
+++ b/mlir/lib/ExecutionEngine/SparseTensor/File.cpp
@@ -24,7 +24,6 @@
 //===----------------------------------------------------------------------===//
 
 #include "mlir/ExecutionEngine/SparseTensor/File.h"
-#include "llvm/Support/ErrorHandling.h"
 
 #include <cctype>
 #include <cstring>
@@ -109,7 +108,6 @@ bool SparseTensorFile::canReadAs(PrimaryType valTy) const {
     // integer and floating primary-types.
     return isRealPrimaryType(valTy);
   }
-  llvm_unreachable("unknown ValueKind");
 }
 
 /// Helper to convert C-style strings (i.e., '\0' terminated) to lower case.
